corehq/apps/registration/utils.py

```diff
@@ -195,7 +195,7 @@ def request_new_domain(request, form, org, domain_type=None, new_user=True):
         dom_req.activation_guid = uuid.uuid1().hex
 
     name = form.cleaned_data['hr_name']
-    with CriticalSection(['request_domain_name_{}'.format(name)]):
+    with CriticalSection([u'request_domain_name_{}'.format(name)]):
```

Comment:
You might have to make the same kind of change in the following places to avoid a possible UnicodeDecodeError:
* corehq/apps/app_manager/util.py:484
* corehq/apps/appstore/views.py:265
* corehq/apps/domain/models.py:703
* corehq/apps/users/views/\_\_init\_\_.py:788

Some of those might be UTF8-encoded (in fact I'm sure that last one is), but it might be worth checking, if you haven't already.